km/km_kkm.c

```diff
@@ -81,3 +81,17 @@ int km_kkm_get_xstate(km_vcpu_t* vcpu, kkm_xstate_t* kx)
    }
    return retval;
 }
+
+/*
+ * depending on syscall or out stack adjustment changes
```

Comment:
I think it warrants an explanation why the adjustment is different for these two cases

Reply:
added comments.